<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalOscilloscope</class>
 <widget class="QMainWindow" name="DigitalOscilloscope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnsleDigitalOscilloscope</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RC.qrc">
    <normaloff>:/new/prefix1/RC/Stats2.ico</normaloff>:/new/prefix1/RC/Stats2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,2">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="Plot" name="QwtPlotFrame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>144</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>串口配置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="SerialLabel">
               <property name="text">
                <string>串    口</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="COMComboBox">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="BaudRateLabel">
               <property name="text">
                <string>波 特 率</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="BaudComboBox"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLabel" name="DataBitLabel">
               <property name="text">
                <string>数 据 位</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QComboBox" name="DataBitComboBox"/>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QLabel" name="ParityBitLabel">
               <property name="text">
                <string>校 验 位</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QComboBox" name="ParityBitComboBox"/>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QLabel" name="StopBitLabel">
               <property name="text">
                <string>停 止 位</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QComboBox" name="StopComboBox"/>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QLabel" name="QueryModelLabel">
               <property name="text">
                <string>查询模式</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QComboBox" name="QueryModeComBox"/>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QLabel" name="TimeOutLabel">
               <property name="text">
                <string>超    时</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QSpinBox" name="timeoutBox">
               <property name="suffix">
                <string> ms</string>
               </property>
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="HLed" name="led" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="7" column="1" colspan="2">
              <widget class="QPushButton" name="OpenCloseButton">
               <property name="text">
                <string>打开/关闭串口</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="d_timerWheel" native="true">
            <property name="maximumSize">
             <size>
              <width>144</width>
              <height>86</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="RecvTextBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>71</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_Qt"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/RC/Standby.ico</normaloff>:/new/prefix1/RC/Standby.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/RC/qt-logo.png</normaloff>:/new/prefix1/RC/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/RC/pic.png</normaloff>:/new/prefix1/RC/pic.png</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QFrame</extends>
   <header>plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="RC.qrc"/>
 </resources>
 <connections/>
</ui>
